static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -399,7 +365,7 @@ class GroupDetails extends Component<Props, State> {
     GroupStore.onPopulateReleases(this.props.params.groupId, releases);
   }
 
-  async fetchData(trackView = false) {
+  async fetchData(trackView = true) {
```

Comment:
Are you sure you want to do this? There are a few places that `fetchData` gets called in this file

Reply:
yeah i think i overcorrected on setting the default here - will fix